CMakeLists.txt

```diff
@@ -2445,38 +2451,11 @@ set(CPACK_WIX_PROPERTY_ARPHELPLINK "${CPACK_PACKAGE_HOMEPAGE_URL}")
 set(CPACK_WIX_UI_BANNER "${CMAKE_CURRENT_SOURCE_DIR}/build/wix/images/banner.bmp")
 set(CPACK_WIX_UI_DIALOG "${CMAKE_CURRENT_SOURCE_DIR}/build/wix/images/dialog.bmp")
 
-include(CPack)
-
-if(APPLE AND MACOS_BUNDLE)
```

Comment:
Why did you remove macOS bundle support?

Reply:
Ups, that was not intended, a rebasing issue. I will rebase again.